lambdas/db-migration/README.md

```diff
@@ -0,0 +1,7 @@
+# @cumulus/db-migration-lambda
+
+## Creating a new migration
+
+```sh
+$ knex migrate:make migration_name
```

Comment:
So this is the command you run in this directory to create new migrations?

Reply:
Correct.

Reply:
So I tried it and it doesn't work:

`zsh: command not found: knex`

Reply:
It seems like this assumes you have `knex` globally installed, but I assume that we should be able to use our local depdendency of knex

Reply:
It seems like you have to specify your database connection info somewhere, which makes sense. Is there some reason we're not including our dev config by default as seen in the Knex docs?

```
module.exports = {
  development: {
    client: 'pg',
    connection: { user: 'me', database: 'my_app' }
  },
```

Reply:
We are here: https://github.com/nasa/cumulus/pull/1810/files?file-filters%5B%5D=#diff-69d8e7315b4a49aed756eedad7781a18R5

Reply:
I don't think that helps when you're running the `knex migrate:make migration_name` command. I can't get that command to succeed without putting some connection info in the knexfile.js

Reply:
Fixed here: 5cd0d2efe